2021-10-06 20:33:51,373 INFO    MainThread:29271 [wandb_setup.py:_flush():69] setting env: {}
2021-10-06 20:33:51,373 INFO    MainThread:29271 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-06 20:33:51,373 INFO    MainThread:29271 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-06 20:33:51,373 INFO    MainThread:29271 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-06 20:33:51,374 INFO    MainThread:29271 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211006_203351-2itxlo2q/logs/debug.log
2021-10-06 20:33:51,374 INFO    MainThread:29271 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211006_203351-2itxlo2q/logs/debug-internal.log
2021-10-06 20:33:51,374 INFO    MainThread:29271 [wandb_init.py:init():375] calling init triggers
2021-10-06 20:33:51,374 INFO    MainThread:29271 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/bio_dataset/bio_ner_dataset.csv', 'checkpoint_path': '/root/konst/model_checkpoints/ner_employment', 'model_path': '', 'batch_size': 128, 'epochs': 25, 'start_epoch': 0, 'test_size': 0.05, 'lam': 0, 'patience': 5, 'factor': 0.5, 'use_wandb': True, 'lr': 0.0003, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 4}
2021-10-06 20:33:51,374 INFO    MainThread:29271 [wandb_init.py:init():424] starting backend
2021-10-06 20:33:51,374 INFO    MainThread:29271 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-06 20:33:51,387 INFO    MainThread:29271 [backend.py:ensure_launched():135] starting backend process...
2021-10-06 20:33:51,392 INFO    MainThread:29271 [backend.py:ensure_launched():140] started backend process with pid: 29288
2021-10-06 20:33:51,394 INFO    MainThread:29271 [wandb_init.py:init():429] backend started and connected
2021-10-06 20:33:51,397 INFO    MainThread:29271 [wandb_init.py:init():477] updated telemetry
2021-10-06 20:33:51,397 INFO    MainThread:29271 [wandb_init.py:init():500] communicating current version
2021-10-06 20:33:51,866 INFO    MainThread:29271 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-06 20:33:51,866 INFO    MainThread:29271 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-06 20:33:52,066 INFO    MainThread:29271 [wandb_init.py:init():540] starting run threads in backend
2021-10-06 20:33:52,827 INFO    MainThread:29271 [wandb_run.py:_console_start():1584] atexit reg
2021-10-06 20:33:52,828 INFO    MainThread:29271 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-10-06 20:33:52,829 INFO    MainThread:29271 [wandb_run.py:_redirect():1463] Redirecting console.
2021-10-06 20:33:52,830 INFO    MainThread:29271 [wandb_run.py:_redirect():1519] Redirects installed.
2021-10-06 20:33:52,830 INFO    MainThread:29271 [wandb_init.py:init():565] run started, returning control to user process
2021-10-06 20:53:58,152 INFO    MainThread:29271 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 255
2021-10-06 20:53:58,152 INFO    MainThread:29271 [wandb_run.py:_restore():1526] restore
